README.md

```diff
@@ -62,6 +62,8 @@ OPTIONS:
 
 The `--threshold` flag accepts [multiple representations](https://docs.rs/byte-unit/4.0.12/byte_unit/struct.Byte.html#examples-2), like `10 GB`, `10 GiB`, or `10GB`. On Linux, percentage-based thresholds like `50%` are also supported.
 
+The `--min-age` flag accepts [multiple representations](https://docs.rs/parse_duration/latest/parse_duration/), such as `4 days` or `1 hour`.
```

Comment:
Please link to the docs for the specific version of the package that Docuum uses (2.1.1) in case the behavior changes upstream